tests/messages_tests/test_session.py

```diff
@@ -36,6 +36,13 @@ def get_request(self):
     def stored_messages_count(self, storage, response):
         return stored_session_messages_count(storage)
 
+    def test_repr(self):
+        storage = self.get_storage()
+        request = self.get_request()
+        self.assertEqual(
+            repr(storage),
+            f'SessionStorage(request = {request!r})')
```

Comment:
I would add `__repr__()` to the `BaseStorage` instead of `SessionStorage`, `CookieStorage`, `FallbackStorage `, e.g.
```python
class BaseStorage:
    ...
    def __repr__(self):
        return f'<{self.__class__.__qualname__} request={self.request!r})>'
```
We cannot recreate a storage from its representation that's why we should use `<...>` notation.

You can add a test to the `messages_testsbase.BaseTests`, e.g.
```python
    def test_repr(self):
        request = self.get_request()
        storage = self.storage_class(request)
        self.assertEqual(
            repr(storage),
            f'<{self.storage_class.__qualname__} request=<HttpRequest>>',
        )
```

Reply:
Thank you for your reviewing my code. I'm not sure that I get the point when we should use` <...> `notation when not.

Reply:
You can check the Python [docs](https://docs.python.org/3/reference/datamodel.html?highlight=__repr__#object.__repr__). It is not possible to recreate a `Storage` object with the string representation that's why we should use the `<...some useful description...>` form.